datafusion/physical-expr/src/expressions/mod.rs

```diff
@@ -22,7 +22,7 @@ mod binary;
 mod case;
 mod cast;
 mod column;
-mod dynamic_filters;
+pub mod dynamic_filters;
```

Comment:
This bit has me tripped up. I'm not sure where the right place to put `dynamic_filters` is such that it's public for our internal use in operators but private from the outside world 🤔